docs/devGuide/settingUp.md

```diff
@@ -34,15 +34,24 @@ We recommend the **WebStorm IDE** for working with MarkBind code.
 ## Setting up the dev environment
 
 1. **Fork and clone** the MarkBind repo.
-1. **To bind your cloned version of MarkBind to your console** (instead of the released version of MarkBind), run `npm link` in the `packages/cli` folder of the cloned repo first.
+1. **Bind your cloned version of MarkBind to your console** Inside the cloned folder navigate to the `packages/cli` folder and run `npm link`
+1. **Install dependencies** by running <popover content="Under the hood, this calls `npm ci` and `lerna bootstrap`">`npm run setup`</popover> in the **root folder** of your cloned repo.
+1. **Congratulations!** Now you ready to start modifying MarkBind code.
 
-   <box type="tip" seamless>
+### Common setup questions
 
-   To go back to the released version of MarkBind, run `npm un -g markbind-cli`, followed by `npm i -g markbind-cli`
-   </box>
-1. **Install dependencies** by running <popover content="Under the hood, this calls `npm ci` and `lerna bootstrap`">`npm run setup`</popover> in the **root folder** of your cloned repo.
-  <box type="info" seamless>
-  MarkBind uses <md>[lerna](https://github.com/lerna/lerna)</md>, a popular multi-package development tool, to manage it's dependencies. It is essentially a high level wrapper over node and npm's functionalities
-  </box>
+1. Does MarkBind work with all operating systems?
 
-1. **Congratulations!** Now you ready to start modifying MarkBind code.
+   Yes! We support all operating systems. However, Windows 11 users might have to run `npm install` after `npm run setup` as some packages may have incompatible versions.
+
+1. How does Markbind manage dependencies?
+
+   MarkBind uses [lerna](https://github.com/lerna/lerna), a popular multi-package development tool, to manage it's dependencies. It is essentially a high level wrapper over node and npm's functionalities.
+
+1. How do I move back to the released version of MarkBind?
+
+   To go back to the released version of MarkBind, run `npm un -g markbind-cli`, followed by `npm i -g markbind-cli`.
+
+1. Some of my front-end components are not working as expected even after running `markbind serve -d`.
```

Comment:
Just checking as I wasn't aware on this as much 😅  is it possible for the front-end to still not work as intended after `serve -d`? I'm under the impression that it and `npm run build:web` achieves the same thing.

Reply:
Yeah, I think @tlylt encountered some issues with the front-end side which actually inspired this section of the docs update. I think the issue had something to do with the popover component not working even with -d

Reply:
(@tlylt can shed some light on this?) I just tried to test the popover myself and while yes, the current popover does not work in normal serve because of recent changes, but it works on `serve -d`.

Reply:
@ryoarmanda @ong6 I think the issue I encountered was that I served the docs via `markbind serve` and forgot to add the `-d` option, and that caused some tooltips to stop working locally due to a recent frontend component update. Doing `markbind serve -d` does fix it. I think this Q & A is still worth adding but need to change the question and answer to caution devs about the `-d` option.

Reply:
@tlylt oops looks like some mis-com happened here XD